care/facility/api/viewsets/testing_lab.py

```diff
@@ -0,0 +1,27 @@
+from django_filters import rest_framework as filters
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.viewsets import ModelViewSet
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'rest_framework.viewsets'](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5348729)